Protos/V1/partitions_common.proto

```diff
@@ -19,6 +21,25 @@ message PartitionRaw {
   int64 priority = 6; /** The priority of the partition. */
 }
 
+/**
+ * Represents every available field in a partition.
+ */
+enum PartitionRawField {
+  PARTITION_FIELD_UNSPECIFIED = 0; /** Unspecified. */
+  PARTITION_FIELD_ID = 1; /** The partition ID. */
+  PARTITION_FIELD_PARENT_PARTITION_ID = 2; /** The parent partition ID. */
```

Comment:
It was plural (IDS)

Reply:
And they should be called `PARTITION_RAW_FIELD_*` or the enum should be `PartitionField `